meshroom/core/node.py

```diff
@@ -1415,10 +1415,19 @@ def __init__(self, nodeType, position=None, parent=None, uids=None, **kwargs):
             for uidIndex in attr.attributeDesc.uid:
                 self.attributesPerUid[uidIndex].add(attr)
 
-        self.optionalCallOnDescriptor("onNodeCreated")
+        try:
+            self.optionalCallOnDescriptor("onNodeCreated")
```

Comment:
Maybe better to directly try/except in optionalCallOnDescriptor.

Reply:
Done